packages/angular/src/schematics/application/application.spec.ts

```diff
@@ -297,6 +297,21 @@ describe('app', () => {
         getFileContent(tree, 'apps/my-dir/my-app/src/app/app.component.ts')
       ).toContain('Thank you for using and showing some ♥ for Nx.');
     });
+
+    it('should update the AppComponent spec to target Nx content', async () => {
```

Comment:
Why is needed after the upgrade?

Reply:
It was due to the changes to the HTML template for generating an application using the Angular schematics. We were just using the generated spec file, but the selectors are no longer valid.